Makefile

```diff
@@ -301,6 +301,10 @@ ifeq ($(BUILD_C_ANALYZERS),yes)
 byte src_build src_build_common test_build: clang_plugin
 endif
 
+ifneq ($(NINJA),no)
+FCP_COMPILE_ARGS = "--ninja --sequential-link"
```

Comment:
I think these quotes make this look like a single argument to the fcp setup script and should be removed. Does it work for you? It complains about the arguments on my machine.

Reply:
I'll just amend the commit, no worries.

Reply:
Yep, it works for me -- what platform? I tested this on Linux with GNU Make 4.2.1, so maybe it is that?

I have a `port` installed version of GNU Make on macOS (4.3); but, by default, macOS appears to have `3.81` (at least, that's what's in my `/usr/bin`).

Reply:
Fails on Debian bullseye and CentOS 7. Of course ninja has to be available to hit the bug. Removing the quotes fixes it.

Reply:
Fair enough ... want me to do it, or is it already done "from your end"?

Reply:
Already done.